data/views/node.2.0.json

```diff
@@ -1,4 +1,13 @@
 {
+<%
+    var _ = injector.get('_');
```

Comment:
@yyscamper: These are views, and we don't have unit-test for them. And, yes, @brianparry indicated that we could remove injector code here. We can add a story to our backlog to revist this file. The intent was to add a ref to obms model for a particular node. I can add a story to ice-box to re-visit this file for clean-ups.